frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewListItem.scss

```
@@ -0,0 +1,7 @@
+.odc-revision-deployment-list {
+    padding: 15px 15px 0px 20px;
```

```suggestion
    padding: 15px 20px 0;
```
Padding right and left: 20px, top: 15px; and bottom: 0;